src/components/charts/CandleChartSimple.vue

```diff
@@ -0,0 +1,582 @@
+<template>
+  <div class="d-flex flex-grow-1 chart-wrapper">
+    <v-chart v-if="hasData" ref="candleChart" :theme="theme" autoresize manual-update />
```

Comment:
why would we need to duplicate the whole chart-component (tree) - instead of simply using what's there already?

That'll be a maintenance nightmare - so unless there's VERY good reasons to do so (it's unclear to me what the differences are - at a glance it seems to be "mostly" copy/pasted) - I'd prefer to keep one chart component only.

Reply:
> why would we need to duplicate the whole chart-component (tree) - instead of simply using what's there already?
The code is roughly written to for the feature demonstration and feedback collection only.

Reply:
well but what's the difference to the original component (so what is "simpler" here??)

Reply:
The simplified labels, legends, and zooming possibilities are removed, so the chart gets more space, and a small green triangle(trade entry) is more visible.

Reply:
i understand labels / legends to some degree - but zooming should remain there (otherwise how would you imagine this to work on webserver mode - where you potentially have 10_000nds of candles? you'd not see anything anymore as you can't zoom.

I don't see this as separate components though (the change is not enough to justify a complete code duplication) - if we need this - then it can be properties on the existing component which disables these functionalities.

Reply:
>  if we need this - then it can be properties on the existing component which disable these functionalities.
Sure, that was an alternative option I preferred not to follow right from the beginning.

Reply:
It might be wrong to add more features to finished and stable software - totally understand that.

I was thinking I've spotted some gaps in UX, so I wanted to address them with this PR. 
I'm ready to finalize the work if needed or close the PR which is also totally fine.